src/python/k4a/docs/building.md

```diff
@@ -0,0 +1,123 @@
+# Building Azure Kinect Python API(K4A)
+
+This information is for how to build your own copy of the Python wheel. 
+If you need to build from source, you have to clone the repository to ensure all the submodule dependencies in place.
+
+## Support Configurations
+
+Python:
+* Greater than or equal to version 3.6.
+
+Architectures: 
+* amd64
+* x86 (limited, no testing is performed)
+
+OS:
+* Windows
+* Linux
+
+## Dependencies
+
+Building on Windows and Linux requires an active internet connection. Part of the
+build will require downloading dependencies.
+
+The following dependencies are for both Windows and Linux:
+
+* [python3](https://www.python.org/getit/). During the install make sure to add
+  python to path.
+  
+* [k4a library](../../../../docs/building.md)
+  The k4a library can be built from the repository or the SDK can be downloaded
+  for the binary. The Windows library is k4a.dll, and the Linux library is k4a.so.
+  The k4a binary needs to be copied to the host system and added 
+  to the path src/python/k4a/src/k4a/_libs in this repository before building.
+
+* [Depth Engine](../../../../docs/depthengine.md). 
+  The depth engine (DE) is a closed source binary shipped with the
+  SDK installer. The DE binary needs to be copied to the host system and added 
+  to the path src/python/k4a/src/k4a/_libs in this repository before building.
+
+The following tools are optional:
+
+* [doxygen](https://github.com/doxygen/doxygen)
+  Required for building documentation.
+
+
+## Building
+
+### Building using a powershell script (Windows)
+
+1. Copy the k4a and DE binaries into the folder src/python/k4a/src/k4a/_libs.
+   The file names MUST be k4a.dll and depthengine.dll. 
+   
+   Note: To create a cross-platform wheel, the .dll for Windows and the .so for Linux need to
+         copied into the folder.
+   Note: If the dlls have different names, create a symlink with relative redirection. This may
```

Comment:
Add a \ for a new line, right now these notes all get thrown together

Reply:
Please fix.

Reply:
Fixed these formatting issues here and in all the other markdowns.